static/js/views/rule-screen.js

```diff
@@ -310,6 +283,11 @@ const RuleScreen = {
       return elt.parentNode;
     }).filter(isValidSelection);
 
+    if (triggerBlocks.length === 0 || effectBlocks.length === 0) {
+      console.log('WHY', triggerBlocks, effectBlocks);
```

Comment:
Haha probably remove the log.

Reply:
Ohhh, I saw that during my sweep for console.logs but I thought it was there already, woops